### ISSUE TITLE
How Does Batch Normalization Help Optimization?

### ISSUE THREAD

**Issue by tkazusa** [tkazusa/papers#30]
https://arxiv.org/pdf/1805.11604.pdf

**The smoothing effect of BatchNorm**
- Indeed, we identify the key impact that BatchNorm has on the training process: it reparametrizes
the underlying optimization problem to make its landscape significantly more smooth. 

Batch Normalizationは有効だがそれは共変量シフトを抑えるからじゃなくて、正規化により出力に対する勾配の変動を抑制し滑らかにする点を主張。BNを導入したネットワークに意図的に共変量シフト(レイヤ出力の平均/分散を変動)させても性能に変化がないことを確認(=そもそも共変量シフトは性能に影響ない)。
